graviton/dryrun.py

```diff
@@ -44,17 +40,51 @@ def _assert_in(status, msgs, msg=None, enforce=True):
     return ok, result
 
 
-def assert_pass(txn_index, msg, txns_res):
-    assert_status("PASS", txn_index, msg, txns_res)
+def assert_error(drr, contains=None, txn_index=None, msg=None, enforce=True):
```

Comment:
re-organizing methods and functions into non-deprecated and deprecated groupings

Reply:
@tzaffi Is there a reason the PR renames methods rather than using Python's deprecation system (https://docs.python.org/3/library/exceptions.html#DeprecationWarning)?

I'm not looking to make or break the PR RE: deprecation style, but as a reviewer I feel obligated to ask.

Alternatively - Since the PR explains the code in question is orphaned, we can skip deprecation + delete here.  The PR already does so for `Invariant.inputs_and_invariants`.

Reply:
I will just delete the code!

Reply:
deleted: 44af893728cb1b1eec96a88a6259ad0b6ab8f100

Reply:
and here: https://github.com/algorand/graviton/pull/42/commits/bf245dd8f636f01ae4a113c68cc5c9b753e2fead

Reply:
@tzaffi Resolving - Thanks for taking in feedback on deprecated methods.  If only we could _always_ delete so freely!